data/core/commandview.lua

```diff
@@ -143,17 +145,25 @@ function CommandView:submit()
 end
 
 
-function CommandView:enter(text, submit, suggest, cancel, validate, wrap_on_overflow)
+function CommandView:enter(text, ...)
```

Comment:
would be nice to have some emmy lua here documenting the options that can be given to the table